source-mysql/main.go

```diff
@@ -24,7 +24,7 @@ const minimumExpiryTime = 7 * 24 * time.Hour
 
 func main() {
 	fixMysqlLogging()
-	var schema = jsonschema.Reflect(&Config{})
+	var schema = schemagen.GenerateSchema("MySQL Source Spec", &Config{})
 	schema.Title = "MySQL Source Spec"
```

Comment:
Setting `schema.Title` here is made redundant by the `title` parameter of `GenerateSchema`